Cryptomator.xcodeproj/xcshareddata/xcschemes/Cryptomator.xcscheme

```diff
@@ -70,9 +70,6 @@
             ReferencedContainer = "container:Cryptomator.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
-      <StoreKitConfigurationFileReference
-         identifier = "../../StoreKitTestConfiguration/Configuration.storekit">
-      </StoreKitConfigurationFileReference>
```

Comment:
Please revert this. I think it's quite useful to have to have this enabled by default.